Makefile

```diff
@@ -8,8 +8,17 @@ export PROJECT_DIR:=$(shell dirname $(realpath $(lastword $(MAKEFILE_LIST))))
 export ADI_DIR:=${PROJECT_DIR}/adi
 export ROOT_DIR:=$(shell dirname "${PROJECT_DIR}")
 
+.PHONY: build test
+
 build:
-	bash ${ADI_DIR}/build.sh
+	echo "${VERSION}" > VERSION.txt
+	pip install -r ${PROJECT_DIR}/requirements/test.txt
 
 test:
 	bash ${ADI_DIR}/test.sh
+
+release:
+	bash ${ADI_DIR}/release.sh
+
+version:
+	echo "1.0.${BUILD_IDENTIFIER}"
```

Comment:
I see you are leaving the setting up of a virtual environment up to the developer. Should this be mentioned in the docs? Maybe it is, I haven't checked.